spec/modules/batch_spec.rb

```diff
@@ -70,6 +70,30 @@
       expect(BatchedSizeWorker).to have_enqueued_job([["bar"], ["bar"], ["bar"]])
       expect(batch.size).to eq(7)
     end
+
+    it 'must put all jobs at once to worker queue' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.